### DISCUSSION TITLE
milvus python sdk - Is it important to open/close connection for each query?

### DISCUSSION THREAD

**Discussion by martinenkoEduard** [milvus-io/milvus#18390] [Q&A and General discussion]
milvus python sdk - Is it important to open/close connection for each query? 
Does milvus have something like transactions?

**Comment by yhmo** [comment]
One connection for one thread/process is recommended. 
If you didn't call close(), the connection will be automatically after a short time about 55 seconds(in python sdk).
Milvus has no transactions.
